diagnostic.md

```diff
@@ -3,67 +3,60 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#readme > article > p:nth-child(1) > a > img
+#user-content-jquery-dom-diagnostic
+#user-content-prerequisites
+#readme > article > div
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('li > OL > BODY');
+$('li');
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('#listerItem').appendTo('form');
```

Comment:
A couple of ways are `$('li').parent().parent().find('form')` and `$('li').parent().parent().children('form');`.